doc/faq.rst

```diff
@@ -131,6 +131,20 @@ For encrypted mailing lists like `schleuder
 Help welcome.
 
 
+How can I participate in Autocrypt?
+-----------------------------------
+
+First of all it would be nice if you join the discussion on IRC.
+Autocrypt lives in #autocrypt on irc.freenode.net, if you have ideas,
+discuss them there first.
```

Comment:
i wouldn't make a hierarchy here but simply say "To ask questions or make suggestions, you can join the ML or join the more informal chat in the #autocrypt room on irc.freenode.net".